### ISSUE TITLE
View Logs Console Bug

### ISSUE THREAD

**Issue by stardemo** [rancher/dashboard#5984]
**Setup**
- Rancher version: v2.6.5  (HA install in aliyun ack) 
- Browser type & version: Chrome v101 

**Describe the bug**

Open Pod(Deployment) View Logs Console and change the log range,
 then close console and open again ,the control bar will disapper.

**To Reproduce**

- Open Pod(Deployment) View Logs Console 
- Change the log range
<img width="625" alt="image" src="https://user-images.githubusercontent.com/20634646/169186186-e040a0a5-499c-4a05-bd64-f8adbbb61e9a.png">

- Close console and refresh page
- Open log console again 

**Result**

- Log console control bar disapper,and log cannot be display.
<img width="821" alt="image" src="https://user-images.githubusercontent.com/20634646/169186627-3a04531b-f188-440f-97c5-0a6557aa1f63.png">

- Brower console error
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/20634646/169186957-e75ca073-7f62-4000-9685-62f331556ab9.png">

**Expected Result**

- Logs console functional.
<img width="679" alt="image" src="https://user-images.githubusercontent.com/20634646/169187156-cb7e1480-e0c1-4710-b78a-f1057b027856.png">

**Temporary Fix**

Use `kubectl` in rancher ha cluster clear user preferences logs-range setting,
```shell
kubectl delete preferences.management.cattle.io -n u-xxxxxx logs-range
```

May be the value of logs-range  storage error,my value is 
<img width="497" alt="image" src="https://user-images.githubusercontent.com/20634646/169187730-37813000-af00-4726-a027-29614b9838c8.png">

**Comment by michael81877** [comment]
We are also seeing this. +1

**Comment by catherineluse** [comment]
# QA Template
## Root cause
I introduced this bug in this PR https://github.com/rancher/dashboard/pull/5850 in which I made the LabeledInput component emit an event that was used by the new SimpleSecretSelector component. The event was named `input`, which was the same name as an event that other components were listening for, so it triggered too many events with this bug as a side effect.
 
## What was fixed, or what changes have occurred
I changed the name of the event from 'input' to 'selecting' so that it will no longer trigger the unwanted 'input' events.
 
## Areas or cases that should be tested
Same as the steps to reproduce the issue
 
## What areas could experience regressions?
I changed the secret selector component so that it listens for the 'selecting' event instead of 'input'. I tested that in my local environment and you can still select secrets for the receivers in AlertmanagerConfigs. But I would say QA probably doesn't need to set up project monitoring just to check for this regression because the secret selectors will need to be retested as part of the fix for this https://github.com/rancher/dashboard/issues/5875 issue anyway.
 
## Are the repro steps accurate/minimal?
I believe so.

**Comment by AbelTaylor** [comment]
@stardemo 
Your temporary solution doesn't seem to work for me here
https://user-images.githubusercontent.com/71317399/170816365-ed1fe981-df11-4e4a-b7f8-c54e59693799.png
https://user-images.githubusercontent.com/71317399/170816375-ca9f41f1-d8fe-4735-8606-205f2dad6e7a.png

**Comment by stardemo** [comment]
@AbelTaylor  you should use `kubectl` in the cluster which run rancher server,if you use rancher server run in docker ,you should use `docker exec ` into rancher container then run `kubectl get ns` to check use ns in k3s storage.

**Comment by AbelTaylor** [comment]
@stardemo

got it , it works, thanks

**Comment by sowmyav27** [comment]
**On 2.6.5 reproduced the issue:**
<img width="1450" alt="Screen Shot 2022-06-03 at 2 10 20 PM" src="https://user-images.githubusercontent.com/26032343/171953389-c48fd5d2-60f3-4202-a3d9-ca188400c455.png">

<img width="344" alt="Screen Shot 2022-06-03 at 2 11 06 PM" src="https://user-images.githubusercontent.com/26032343/171953435-7f63af7d-2834-43f0-91ee-19b8a85aa43f.png">

On 2.6head commit id: `79a3bcb`

- View logs pop up is blank. No logs are displayed.
- Error in console

```
vue.runtime.esm.js:1897 TypeError: r.replace is not a function
    at C (ResourceCancelModal.vue:153:1)
    at f.rangeOptions (ResourceCancelModal.vue:116:9)
    at Tn.get (vue.runtime.esm.js:4495:25)
    at Tn.evaluate (vue.runtime.esm.js:4597:21)
    at f.rangeOptions (vue.runtime.esm.js:4851:17)
    at fn (ResourceCancelModal.vue?e977:1:2097)
    at Object.r [as title] (vue.runtime.esm.js:2605:62)
    at f.<anonymous> (blank.vue?49d9:1:159)
    at t._render (vue.runtime.esm.js:3569:22)
    at f.r (vue.runtime.esm.js:4081:21)
re @ vue.runtime.esm.js:1897
vue.runtime.esm.js:1897 TypeError: Cannot read properties of undefined (reading 'addEventListener')
    at ResourceCancelModal.vue:222:1
    at d (runtime.js:45:40)
    at Generator._invoke (runtime.js:274:22)
    at forEach.t.<computed> [as next] (runtime.js:97:21)
    at r (asyncToGenerator.js:3:20)
    at l (asyncToGenerator.js:25:9)vue.runtime.esm.js:1897 TypeError: r.replace is not a function
    at C (ResourceCancelModal.vue:153:1)
    at f.rangeOptions (ResourceCancelModal.vue:116:9)
    at Tn.get (vue.runtime.esm.js:4495:25)
    at Tn.evaluate (vue.runtime.esm.js:4597:21)
    at f.rangeOptions (vue.runtime.esm.js:4851:17)
    at fn (ResourceCancelModal.vue?e977:1:2097)
    at Object.r [as title] (vue.runtime.esm.js:2605:62)
    at f.<anonymous> (blank.vue?49d9:1:159)
    at t._render (vue.runtime.esm.js:3569:22)
    at f.r (vue.runtime.esm.js:4081:21)
re @ vue.runtime.esm.js:1897
vue.runtime.esm.js:1897 TypeError: Cannot read properties of undefined (reading 'addEventListener')
    at ResourceCancelModal.vue:222:1
    at d (runtime.js:45:40)
    at Generator._invoke (runtime.js:274:22)
    at forEach.t.<computed> [as next] (runtime.js:97:21)
    at r (asyncToGenerator.js:3:20)
    at l (asyncToGenerator.js:25:9)
```

**Comment by richard-cox** [comment]
@sowmyav27 There's a connected issue that should be tested alongside this one - #6122. The associated change contains a fix to update existing, broken settings to ensure effected users don't have to apply the workaround of deleting it.

**Comment by xhejtman** [comment]
Version 2.6.6 has been released and the bug is still present :(

**Comment by sowmyav27** [comment]
As stated [here](https://github.com/rancher/rancher/releases/tag/v2.6.6), "Rancher v2.6.6 is a mirror release of [v2.6.5](https://github.com/rancher/rancher/releases/tag/v2.6.5) with one backend change". There are no UI fixes in 2.6.6.

**Comment by papanito** [comment]
Is there an ETA when this fix will be released?

**Comment by richard-cox** [comment]
@papanito The issue is fixed in v2.6.7
